test/python/provider-upgrade-command/cdktf.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
I kept this in line with the provider add test, we can do a quick follow up to change both, but I don't want to do it here since this PR keeps on dragging for too long already